catalog/core/catalog-core-migratable/src/main/java/org/codice/ddf/catalog/migratable/impl/CatalogMigratableConfig.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.migratable.impl;
+
+import java.nio.file.Path;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Contains configuration constants used to change the behavior of catalog migratable operations.
+ */
+public class CatalogMigratableConfig {
+
+    private static final int MAX_CARDS_PER_FILE = 50000;
```

Comment:
They could be -- but then an administrator might accidentally break `CatalogMigratableConfig` and be shut out from ever exporting. There are a few key relationships that need to be preserved (see getter's / setters in this class and the `exportMetacardQuery` method in `MigrationTaskManager`. 

Should we perhaps widen the range of these maxima and minima?